config/webpack.test.js

```diff
@@ -1,76 +1,32 @@
 /* eslint strict: 0 */
 'use strict';
 
-const common = require('./webpack.common');
 const path = require('path');
+const common = require('./webpack.common');
+const externals = {
+  jquery: 'jQuery',
+};
 
-module.exports = {
-
-  mode: 'production',
-
-  optimization: {
-    minimize: false
-  },
-
-  devtool: 'source-map',
-
-  entry: './test/test.js',
-
-  output: {
-    path: path.resolve('./build/test'),
-    pathinfo: true,
-    filename: 'test.bundle.js'
-  },
-
-  resolve: {
-    modules: ["node_modules"],
-    alias: {
-      'emailjs-stringencoding': path.resolve('./src/lib/string-encoding'),
-      'text-encoding': path.resolve('./src/lib/string-encoding')
-    }
-  },
-
-  externals: {
-    jquery: 'jQuery'
-  },
+const resolve = {
+  modules: [path.resolve('./src'), 'node_modules'],
+  alias: {
+    'mailreader-parser': path.resolve('./node_modules/mailreader/src/mailreader-parser'),
```

Comment:
mailreader is no longer a dependency and has been integrated into our code base now, so this line is obsolete.